dictionaries/admin.translation.json

```diff
@@ -885,7 +885,6 @@
 		"sr": "Shib 1.3 Davalac Identiteta (udaljeni)",
 		"ro": "Furnizor de identitate Shib 1.3 (distant)",
 		"ru": "\u041f\u0440\u043e\u0432\u0430\u0439\u0434\u0435\u0440 \u0438\u0434\u0435\u043d\u0442\u0438\u0444\u0438\u043a\u0430\u0446\u0438\u0438 Shib 1.3 (\u0443\u0434\u0430\u043b\u0435\u043d\u043d\u043e\u0435 \u0440\u0430\u0437\u043c\u0435\u0449\u0435\u043d\u0438\u0435)",
-		"eu": "Shib 1.3 Identitate hornitzailea (Urrunekoa)",
```

Comment:
In el translation, some of character should not be translated. Such as '@' \u0040, ';' \u003b, and many symbols.
In dictionaries/logout.translation.json:594, there have one translation forgot to encode.
I do no change any content of other language, except zh-tw. Of other language, I only corrected all of data to same format using some decode and encode tools.